### PR TITLE
Alternative perception module, fully working. Also I'm Batman. 

### PR DESCRIPTION
**Pull request by wAuner**
ok fellas, I also worked on an perception module. This includes a new classifier class and I also trained a model. Although I wasn't able to finish training due to some weird tensorflow error, the model in combination with the new class, was good enough to get everything working without errors for the first time. I use the same model for real world and simulator deployment.

On a very small test set with ~100 images from real world and sim images I get 100% accuracy. 

Testing in the simulator works perfectly without any errors. Fixes also #17. In doubt, activate the camera a little earlier before activating autopilot. Even previously critical situations like changing traffic lights have been handled safely during testing.

Testing with both rosbag files worked without errors.

Feel free to test and compare with @stevendae 's model, it integrates with plug and play in my new classifier logic. Integrate whatever works best.

My trained inference graph can be downloaded here:
Edit: see link in the comment below

Just make sure not to confuse the different models, because the filename is the same.

I will post the password in slack.

### CODE REVIEW

**Comment by wAuner** [comment]
I've set up a new production environment and exported the graph again to verify that it works under production conditions. Took quite some time, but now it seems to work. Download the new graph 


# --------------->[here](https://1drv.ms/u/s!AtnHmLaNdjwXibY3GmBTQgUOMpuzMA)<----------------------

Password stays the same.

I ran into an issue with the pillow version on the production system, which was also mentioned on the forums. I could only receive image data from the simulator with a newer version of pillow and not with the one from requirements.txt. But this maybe due to some indirect dependencies.
